### DISCUSSION TITLE
create webp format when uploading jpg or png

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#45210] [Q&A] [answered]
I want to automatically create `webp` format when uploading `jpg` or `png` photo and save it on the server. How easy is it to do this in Symfony?

**Comment by michaoj** [comment, marked as answer]
This is not related to symfony but you could use the GD library and the function imagewebp to achieve your goal

https://www.php.net/manual/fr/function.imagewebp.php

Once you have your image uploaded on your server you can do anything you want on it
